Build.PL

```diff
@@ -19,6 +19,10 @@ my $class = npg_tracking::util::build->subclass(code => <<'EOF');
     'gt_utils' => {
       'copy'     => [qw/find_gt_match gt_pack/],
                   },
+    'fastqcheck' => {
+      'copy'     => [qw/fastqcheck/],
+      'samtools' => 1,
+                  },
```

Comment:
This addition should be moved under the 'if (which('lsb_release')' clause. We need to ask @keithj to look at it and, most likely, add npg_conda.